Kotlin/hello-world.kt

```diff
@@ -0,0 +1,7 @@
+// This is the main function
```

Comment:
As a side note, I did not see in the Java file instructions to a newbie on how to compile and run a java class. This isn't the end of the world, but overall someone should consider adding documentation or links on how to do this.

Reply:
Yeah, thanks for the suggestion but given that the repository focuses on hello world programs in as many languages as possible, it is not practically possible to add instructions for every language. You can always search online for help for installation or any other issue as there are a great amount of resources available for that.